scripts/sql-schemacompare-test-integration.bat

```diff
@@ -0,0 +1,10 @@
+setlocal
+
+set ADS_TEST_GREP=Schema compare integration test suite
```

Comment:
Don't think this should be merged in. If anything make a script like Benjin did for the extension unit tests that can be used to specify which tests to run